test/storages/sequelize.test.ts

```diff
@@ -0,0 +1,408 @@
+				.then(allTables => {
+					expect(allTables).toEqual(['SequelizeMeta']);
+				});
+		});
+
+		it('deletes the migration from the database', () => {
+			const storage = new Storage({ sequelize: helper.sequelize });
+
+			return storage
+				.logMigration('asd.js')
+				.then(() => storage.model.findAll())
+				.then(migrations => {
+					expect(migrations.length).toBe(1);
+				})
+				.then(() => storage.unlogMigration('asd.js'))
+				.then(() => storage.model.findAll())
+				.then(migrations => {
+					expect(Object.keys(migrations)).toHaveLength(0);
+				});
+		});
+
+		it('deletes only the passed migration', () => {
+			const storage = new Storage({ sequelize: helper.sequelize });
+
+			return storage
+				.logMigration('migration1.js')
+				.then(() => storage.logMigration('migration2.js'))
+				.then(() => storage.unlogMigration('migration2.js'))
+				.then(() => storage._model().findAll())
+				.then(migrations => {
+					expect(migrations.length).toBe(1);
+					expect(migrations[0].name).toBe('migration1.js');
+				});
+		});
+
+		it('deletes the migration from the database with a custom column name', () => {
+			const storage = new Storage({
+				sequelize: helper.sequelize,
+				columnName: 'customColumnName',
+			});
+
+			return storage
+				.logMigration('asd.js')
+				.then(() => storage.model.findAll())
+				.then(migrations => {
+					expect(migrations.length).toBe(1);
+				})
+				.then(() => storage.unlogMigration('asd.js'))
+				.then(() => storage.model.findAll())
+				.then(migrations => {
+					expect(Object.keys(migrations)).toHaveLength(0);
+				});
+		});
+
+		it('deletes the migration from the database with timestamps', () => {
+			const storage = new Storage({
+				sequelize: helper.sequelize,
+				timestamps: true,
+			});
+
+			return storage
+				.logMigration('asd.js')
+				.then(() => storage.model.findAll())
+				.then(migrations => {
+					expect(migrations.length).toBe(1);
+				})
+				.then(() => storage.unlogMigration('asd.js'))
+				.then(() => storage.model.findAll())
+				.then(migrations => {
+					expect(Object.keys(migrations)).toHaveLength(0);
+				});
+		});
+	});
+
+	describe('executed', () => {
+		it("creates the table if it doesn't exist yet", () => {
+			const storage = new Storage({
+				sequelize: helper.sequelize,
+			});
+
+			return storage.model.sequelize
+				.getQueryInterface()
+				.showAllTables()
+				.then(allTables => {
+					expect(Object.keys(allTables)).toHaveLength(0);
+				})
+				.then(() => storage.executed())
+				.then(() => storage.model.sequelize.getQueryInterface().showAllTables())
+				.then(allTables => {
+					expect(allTables).toEqual(['SequelizeMeta']);
+				});
+		});
+
+		it('returns an empty array if no migrations were logged yet', () => {
+			const storage = new Storage({
+				sequelize: helper.sequelize,
+			});
+
+			return storage.executed().then(migrations => {
+				expect(Object.keys(migrations)).toHaveLength(0);
```

Comment:
Just out of curiosity - wouldn't `expect(migrations).toEqual({})` work?

Reply:
Yes it probably would - this file is a straight up port, I just ran jest-codemods and cleaned up some of the context variables, I didn't touch any of the assertions.